pom.xml

```diff
@@ -60,6 +60,7 @@
         <commons-collections4.version>4.4</commons-collections4.version>
         <commons-csv.version>1.11.0</commons-csv.version>
         <commons-lang3.version>3.16.0</commons-lang3.version>
+        <odfdom.version>0.9.0</odfdom.version>
```

Comment:
just aware this is the latest version based on JDK 8 and it published at 2021... so any alternative solution?

Reply:
> just aware this is the latest version based on JDK 8 and it published at 2021... so any alternative solution?

Previously, using version 0.12.0 required JDK 11. To ensure compatibility with JDK 8, it was downgraded to version 0.9.0.

Reply:
just wonder if any alternative solution as odfdom won't pin on JDK 8 as POI.